.eslintrc

```diff
@@ -0,0 +1,157 @@
+/* Based on https://github.com/FreeCodeCamp/freecodecamp/blob/staging/.eslintrc */
```

Comment:
Can you please use [wealthfront-javascript](https://github.com/wealthfront/javascript) as the rules?

You can see an example of how to hook that up in this project: https://github.com/wealthfront/privatestate/blob/master/.eslintrc